var/spack/repos/builtin/packages/linaro-forge/package.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import platform
+import subprocess
+
+from spack.package import *
+from spack.version import StandardVersion
+
+
+class LinaroForge(Package):
+    """Build reliable and optimized code for the right results on multiple
+    Server and HPC architectures, from the latest compilers and C++ standards
+    to Intel, 64-bit Arm, AMD, OpenPOWER and Nvidia GPU hardware. Linaro Forge
+    combines Linaro DDT, the leading debugger for time-saving high performance
+    application debugging, Linaro MAP, the trusted performance profiler for
+    invaluable optimization advice across native and Python HPC codes, and
+    Linaro Performance Reports for advanced reporting capabilities."""
+
+    homepage = "https://www.linaroforge.com"
+    maintainers("kenche-linaro")
+
+    if platform.machine() in ["aarch64", "arm64"]:
+        version(
+            "23.0.3", sha256="a7e23ef2a187f8e2d6a6692cafb931c9bb614abf58e45ea9c2287191c4c44f02"
+        )
+        version(
+            "23.0.2", sha256="698fda8f7cc05a06909e5dcc50b9956f94135d7b12e84ffb21999a5b45c70c74"
+        )
+        version(
+            "23.0.1", sha256="552e4a3f408ed4eb5f1bfbb83c94530ee8733579c56c3e98050c0ad2d43eb433"
+        )
+        version("23.0", sha256="7ae20bb27d539751d1776d1e09a65dcce821fc6a75f924675439f791261783fb")
+        version(
+            "22.1.4", sha256="4e2af481a37b4c99dba0de6fac75ac945316955fc4170d06e321530adea7ac9f"
```

Comment:
Do you mean `"22.0.4"` here?

Reply:
This is the blocker fix for me since I get an error trying to fetch 22.1.4.

Reply:
This should be resolved now, thanks.